uaclient/entitlements/fips.py

```diff
@@ -535,8 +547,16 @@ def messaging(self) -> MessagingOperationsDict:
 
     def _perform_enable(self, silent: bool = False) -> bool:
         if super()._perform_enable(silent=silent):
-            self.cfg.notice_file.try_remove(
-                "", messages.FIPS_DISABLE_REBOOT_REQUIRED
+            notices.remove(
```

Comment:
Same remove consideration

Reply:
Makes sense.

Reply:
I think this comment still applies and this remove call can be removed @CalvoM